docs/resources/vpc_firewall_rules.md

```diff
@@ -6,9 +6,49 @@ description: |-
   This resource manages VPC firewall rules.
   !> Firewall rules defined by this resource are considered exhaustive and will
   overwrite any other firewall rules for the VPC once applied.
-  !> Setting the rules attribute to [] will delete all firewall rules for the
+  !> Setting the rules attribute to {} will delete all firewall rules for the
   VPC which may cause undesired network traffic. Please double check the firewall
   rules when updating this resource.
+  Migrating rules
+  Previous versions of this resource stored firewall rules in a set. This
+  resulted in slow plans in environments with a significant number of rules.
+  Newer versions store the rules in a map for better performance, but this change
```

Comment:
Your screenshot looks good, but did we lose some formatting here? Should this be a numbered list?